simulation.py

```diff
@@ -118,15 +119,35 @@ def simulate(alpha_settings_path, skip_evaluation,
     eval_results = (
         Evaluator().run(TRADING_INSTRUMENTS[settings.target], results)
         if not skip_evaluation else None)
-    handle_result(output_path, skip_screen_print, results, eval_results)
+    results_queue.put((alpha_settings_path, results, eval_results))
 
 def main():
     """
     Run the simulation process.
     """
     args = build_parser()
-    simulate(args.alpha_settings_path, args.skip_evaluation,
-             args.skip_screen_print, args.output_path)
+    data_all = initialize()
+    processes = []
+    results_queue = Queue()
+    for path in args.alpha_settings_paths:
+        process = Process(
+            target=simulate,
+            args=(
+                results_queue, data_all,
+                path, args.skip_evaluation))
+        process.start()
+        processes.append(process)
+    unordered_results = {}
+    while any(process.is_alive() for process in processes):
+        if not results_queue.empty():
+            path, results, eval_results = results_queue.get()
+            unordered_results[path] = (results, eval_results)
+    for process in processes:
+        process.join()
+    for path, (results, eval_results) in unordered_results.items():
```

Comment:
Can we make execution order of handle_result ordered by the user's input order?
For now the result order is not stable due to race condition.